lib/i18n/tasks/translators/openai_translator.rb

```diff
@@ -52,7 +52,14 @@ def no_results_error_message
     private
 
     def translator
-      @translator ||= OpenAI::Client.new(access_token: api_key)
+      @translator ||=
+        if @i18n_tasks.translation_config[:openai_log_errors]
+          OpenAI::Client.new(access_token: api_key, log_errors: @i18n_tasks.translation_config[:openai_log_errors]) do |f|
+            f.response :logger, Logger.new($stdout), bodies: true
```

Comment:
1. Errors should be logged to `STDERR`
2. I think we can always log the errors, no need for the environment variable